fighthealthinsurance/chat_interface.py

```
@@ -14,14 +14,21 @@
 from fighthealthinsurance.ml.ml_router import ml_router
 from fighthealthinsurance.ml.ml_models import RemoteModelLike
-from fighthealthinsurance.models import OngoingChat, Denial, Appeal, PriorAuthRequest
+from fighthealthinsurance.models import (
+    OngoingChat,
+    Denial,
+    Appeal,
+    PriorAuthRequest,
+    ChatLeads,
+)
 from fighthealthinsurance.pubmed_tools import PubMedTools
 from fighthealthinsurance import settings
```

Ruff (0.11.9)

`fighthealthinsurance.settings` imported but unused

Remove unused import: `fighthealthinsurance.settings`

(F401)